### PR TITLE
Add make restart command and make restart-detached command.

### PR DESCRIPTION
**Pull request by MichaelRoytman**
…ersonal make commands

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,7 @@ npm-debug.log
 coverage
 
 dist/
+
+# Personal make commands
+local.mk
```
Comment (fysheets) [inline .gitignore]: Would any of the personal commands that you're adding make sense to be in the overall Makefile? That way we don't have to worry about ignoring them and others could leverage them as well? Or are they very specific to your working environment?

File: Makefile [hunk 2]
```diff
@@ -39,6 +41,10 @@ from-scratch: ## start development environment from scratch
 	docker build -t edxops/studio-frontend:latest --no-cache .
 	make up
 
+restart:
```
Comment (fysheets) [inline Makefile]: 👍

File: .gitignore (hunk 1 above)
Comment (MichaelRoytman) [reply .gitignore]: I added a `make restart-detached` command for myself, which basically just does `up-detached` after the down. I didn't get the sense that anyone really used the up-detached command, so I didn't want to clutter the Makefile with my commands. If people would use it, I'm happy to add it. What do you think?

Comment (fysheets) [reply .gitignore]: I think it's a pretty useful command that would be good to have available for the group in general :) One more command won't clutter it up.

**Comment by MichaelRoytman** [comment]
@fysheets I've added the `make restart-detached` make command and removed the .gitignore changes. Does this look okay?

**Pull request merged**